### PR TITLE
(docs) Update geojson.ipynb

### PR DESCRIPTION
**Pull request by gnestor**
- [x] I have read the [Contributor Guide](https://github.com/nteract/nteract/blob/master/CONTRIBUTING.md)
- [x] My PR title and commit messages are in [conventional-changelog-standard](https://github.com/nteract/nteract/blob/master/CONTRIBUTING.md#how-should-i-write-my-commit-messages-and-pr-titles) format.

* Use `IPython.display.GeoJSON` from ipython 6
* Demonstrate how to configure Leaflet maps using metadata

### CODE REVIEW

File: example-notebooks/geojson.ipynb [hunk 1]
```diff
@@ -1,287 +1,488 @@
...
+			"outputs": [],
+			"source": [
+				"data = {\n",
+				"\t\"type\": \"FeatureCollection\",\n",
+				"\t\"features\": [{\n",
+				"\t\t\"type\": \"Feature\",\n",
+				"\t\t\"properties\": {\n",
+				"\t\t\t\"place\": \"The coffee bar\",\n",
+				"\t\t\t\"login\": \"espresso\",\n",
+				"\t\t\t\"lat\": \"38.91427\",\n",
```
Comment (rgbkrk) [inline example-notebooks/geojson.ipynb]: What's up with these tabs?

File: example-notebooks/geojson.ipynb [hunk 2]
```diff
@@ -1,287 +1,488 @@
...
-   },
-   "name": "python",
-   "pygments_lexer": "ipython3",
-   "version": "3.5.1"
-  }
- },
- "nbformat": 4
-}
\ No newline at end of file
+	"cells": [{
```
Comment (rgbkrk) [inline example-notebooks/geojson.ipynb]: Could you add an intro to this document that says something about GeoJSON and needing IPython 6? Something like:

```
With the release of IPython 6 (coming April 2017), there's a GeoJSON display class that allows you to visualize GeoJSON data quickly.
```

**Comment by rgbkrk** [comment]
Thanks for updating this, it's awesome.

File: example-notebooks/geojson.ipynb (hunk 1 above)
Comment (gnestor) [reply example-notebooks/geojson.ipynb]: Well, easier to read but take up more space...

Comment (rgbkrk) [reply example-notebooks/geojson.ipynb]: I was mostly curious if this was being introduced by nteract in some way. My primary concern is that if someone copies this code they may end up with mixed space types which are a cause of disaster in python.

**Pull request merged**